test/func/packages/html-reporter-test-server/index.js

```diff
@@ -18,17 +18,24 @@ module.exports = (testplane, pluginConfig) => {
     testplane.on(testplane.events.RUNNER_START, () => {
         const app = express();
         app.use(express.static(path.resolve(__dirname, '../..')));
+
         server = app.listen(port, (err) => {
             if (err) {
                 console.error('Failed to start test server:');
                 throw new Error(err);
             }
 
             console.info(`Server is listening on http://localhost:${port}`);
+        }).on('error', err => {
+            if (err.code === 'EADDRINUSE') {
+                console.warn(`Skip running server, because port ${port} is busy`);
+            } else {
+                throw err;
+            }
```

Comment:
- We run Testplane (which uses this html-reporter-test-server and opens the server on SERVER_PORT)
  - Testplane runs Testplane GUI (port GUI_PORT)
    - Testplane GUI uses html-reporter-test-server and tries to run server on SERVER_PORT